### PR TITLE
rgw:"Cannot get slave zonegroup's bucket location in master zonegroup returned 301

### PR DESCRIPTION
**Pull request by snakeAngel2015**
fixes:http://tracker.ceph.com/issues/17975

### CODE REVIEW

**Comment by snakeAngel2015** [comment]
@yehudasa @oritwas

**Comment by snakeAngel2015** [comment]
@yehudasa @oritwas

**Comment by oritwas** [comment]
@snakeAngel2015 please rebase

File: src/rgw/rgw_common.h [hunk 1]
```diff
@@ -401,6 +401,7 @@ enum RGWOpType {
   RGW_OP_STAT_ACCOUNT,
   RGW_OP_LIST_BUCKET,
   RGW_OP_GET_BUCKET_LOGGING,
+  RGW_OP_GET_BUCKET_LOCATION,
```
Comment (amitkumar50) [inline src/rgw/rgw_common.h]: @snakeAngel2015 (NIT) Please consider to keep PR title < 50 characters for future PRs.
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits

**Pull request closed without merging**